src/services/shareSheetHandler.ts

```diff
@@ -12,9 +13,13 @@ export const handleShare = (
     execute
         .then((result) => {
             if (result.data?.createInvitation.id) {
+                const expoLink = Linking.createURL(
+                    "joinTrip/" + result.data?.createInvitation.id,
+                    {}
+                );
+                console.log(expoLink);
```

Comment:
Is this log output actually useful?

Reply:
Depends, i think for testing purposes it was useful, because it was easy to copy the link and add to your mobile phone browser, not too sure if we still need it tough. What do you think @valentingc ?

Reply:
I'd leave it as soon as we don't have it running in production.

Reply:
Works for me! 👍